src/annotator/test/sidebar-test.js

```diff
@@ -599,29 +618,29 @@ describe('Sidebar', () => {
       const { port1 } = new MessageChannel();
       sidebar.onFrameConnected('dummy', port1);
 
-      assert.notCalled(fakeBridge.createChannel);
+      assert.notCalled(sidebarBridge().createChannel);
```

Comment:
You'll also need an `assert.notCalled(guestBridge().createChannel)` check here.

Reply:
That's a detailed catch! Thanks, fixed.